cmd/kube-apiserver/app/aggregator.go

```diff
@@ -250,6 +250,7 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 	{Group: "events.k8s.io", Version: "v1"}:                      {group: 17750, version: 15},
 	{Group: "events.k8s.io", Version: "v1beta1"}:                 {group: 17750, version: 5},
 	{Group: "authentication.k8s.io", Version: "v1"}:              {group: 17700, version: 15},
+	{Group: "authentication.k8s.io", Version: "v1alpha1"}:        {group: 17700, version: 15},
```

Comment:
shouldn't the version (on the far right) be lower than the other group, to ensure that this is properly sorted in the API, ie. `v1alpha1` with lower priority?

Reply:
Yes, it looks like a bug. I changed the version value to `1`.